stl/inc/utility

```diff
@@ -802,6 +818,26 @@ _NODISCARD constexpr underlying_type_t<_Ty> to_underlying(_Ty _Value) noexcept {
     _CSTD abort(); // likely to be called in debug mode, but can't be relied upon - already entered the UB territory
 #endif // _DEBUG
 }
+
+template <class _Ty, class _Uty>
+_NODISCARD constexpr auto&& forward_like(_Uty&& _Ux) noexcept {
+    static_assert(_Can_reference<_Ty>, "std::forward_like's first template argument must be a referenceable type.");
+
+    using _UnrefU = remove_reference_t<_Uty>;

```

Comment:
To make this closer to the wording of the standard, adding `using _UnrefT = remove_reference_t<_Ty>` here would be good - we do a `remove_reference_t` in both cases anyways.

Reply:
Most review comments are addressed. But I prefer to leave the hard error caused by invalid `_Ty&&` in `static_assert`, so that we can control the error message.

Reply:
Oh, to be clear I wasn't attempting to ask you to remove that, sorry!